codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  require_ci_to_pass: true
+
+comment:
+    behavior: default
+    branches:
+        - "master"
+
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 0%
+
+    patch:
+      default:
+        target: 100%
```

Comment:
```suggestion
        target: 100%
        
codecov:
  token: 27bb1e3f-0e2c-4c4e-beca-45f213201f9a
```

Reply:
We shouldn't need this here. It's better to use the `GITHUB_SECRET` which happens by default the way @OriolAbril has it set up, I think. We would only need this for a private repo.

Reply:
codecov should not need any token nor secret for public repos like this one, I'll try rerunning the jobs without any of the two and if it does not work add a secret

Reply:
That's because it's *using* the `GITHUB_SECRET` that is already available by default. No, you shouldn't need to add it manually!